components/financial_assistance/app/domain/financial_assistance/operations/applications/ifsv/h9t/ifsv_eligibility_determination.rb

```diff
@@ -72,7 +72,13 @@ def update_applicant_evidence(applicant, status, response_applicant_entity, enro
                 applicant.set_income_evidence_verified
               when "outstanding"
                 if income_evidence.enrolled_in_any_aptc_csr_enrollments?(enrollments)
-                  applicant.set_evidence_outstanding(income_evidence)
```

Comment:
@hKalesnikavaideacrew what happens when the applicant is not enrolled in aptc/csr and they have evidence in review/rejected? Do we transition them to NRR?

Reply:
Yes. Ticket required it and code implements this logic. 
I specifically asked this from Hithesh